### PR TITLE
Checking Timing of a Program in C

### PR DESCRIPTION
**Pull request by Nibba2018**
Parent issue: #48 

### Type of change

**Select all that applies**:
* [X] New feature (non-breaking change which adds functionality)

### Checklist:

* [X] My code follows the style guidelines(Clean Code) of this project.
* [X] I have performed a self-review of my own code.
* [X] I have used consistent code conventions and clear function/method/variable names.
* [X] I have commented on my code, particularly in hard-to-understand areas.
* [X] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [X] I have added relevent screenshots/images if necessary in a seperate folder.
* [X] I have added my work in respective language folder.
* [X] My changes generate no new warnings.

### CODE REVIEW

File: C/Program-Timer/timer.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+int main(int argc, char *argv[])
+{
+    /*
+    Create start and end clock variables of type `clock_t`
+    and time the function.
+    */
+    clock_t start = clock();
+    function(23, 54);
```
Comment (yashaswibiyahut) [inline C/Program-Timer/timer.c]: use dynamic input

File: C/Program-Timer/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Program Timing
+This script checks the time required to run a function in C.
+
+
+## To use
+* Edit `void function` and write your own operations.
+* Compile with `gcc timer.c -o timer`.
+* Execute with `./timer`
+
+
```
Comment (yashaswibiyahut) [inline C/Program-Timer/README.md:12]: please add space and time complexity.

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

File: C/Program-Timer/README.md (hunk 2 above)
Comment (Nibba2018) [reply C/Program-Timer/README.md:12]: Hello @yashaswibiyahut, I am not sure what you mean by this. Do you want me to find the time and space complexity of `function` programatically?

Comment (yashaswibiyahut) [reply C/Program-Timer/README.md:12]: add simple space and time complexity of your **program**

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

**Review by yashaswibiyahut — APPROVED** [review]

**Pull request merged**